person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
   def initialize(id, age, name: 'Unknown', parent_permission: true)
```

Comment:
- [ OPTIONAL ] I also noticed that the code provided in the project requirements had to be adjusted to properly test your project. This is because the `initialize` method of your `Person` class expects two arguments: `id` and `age`. But in the code provided in the project requirements which I have pasted below it only has the `age` argument and the `name` argument which is optional. So I will suggest that you include the right code for testing your project in your project so it's easier for visitors to test your code :wink: 
```
person = Person.new(22, 'maximilianus')
  person.correct_name
  capitalizedPerson = CapitalizeDecorator.new(person)
  capitalizedPerson.correct_name
  capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
  capitalizedTrimmedPerson.correct_name
```
- [ ] This is the code I used to test your project below. You can paste that in your `person.rb` file or you can create a new file specifically for testing your project. :wink: 
- I also had to import the required files into the `person.rb` so don't forget to do that as well :wink: 

```
person = Person.new(1, 22, name: 'maximilianus')
 puts person.correct_name
 capitalizedPerson = CapitalizeDecorator.new(person)
 puts capitalizedPerson.correct_name
 capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
 puts capitalizedTrimmedPerson.correct_name

```